File: snap-crescent/src/main/java/com/codeinsight/snap_crescent/appConfig/AppConfig.java
package com.codeinsight.snap_crescent.appConfig;

import java.io.Serializable;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "APP_CONFIG")
public class AppConfig implements Serializable {

	private static final long serialVersionUID = -6497139748806438592L;

	@Id
	private long id;

	private String configKey;

	private String configValue;

	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	public String getConfigKey() {
		return configKey;
	}

	public void setConfigKey(String configKey) {
		this.configKey = configKey;
	}

	public String getConfigValue() {
		return configValue;
	}

	public void setConfigValue(String configValue) {
		this.configValue = configValue;
	}
}

File: snapcrescent_mobile/lib/models/app_config.dart
class AppConfig  {
  
  String? configKey;
  String? configValue;

  AppConfig  (
      {this.configKey,
      this.configValue,
      });


  factory AppConfig.fromMap(Map<String, dynamic> map) {
      return AppConfig(
        configKey : map['CONFIG_KEY'],
        configValue : map['CONFIG_VALUE']
      );

   }

  Map<String,dynamic> toMap() {
    Map<String,dynamic> map =  {};

    map['CONFIG_KEY'] = configKey;
    map['CONFIG_VALUE'] = configValue;

    return map;
  }
}
